documentation/src/main/asciidoc/integrationguide/chapters/services/extras/override/ex2-contributor.java

```diff
@@ -1,4 +1,4 @@
-public class LatestAndGreatestConnectionProviderImplContributor1
+public class LatestAndGreatestConnectionProviderImplContributor
         implements ServiceContributor {
```

Comment:
Originally the example code snippets were prepared for two flavours:

- LatestAndGreatestConnectionProviderImplContributor1
- LatestAndGreatestConnectionProviderImplContributor2

one of the reasons is two `meta-inf` files were provided so using number suffix to differentiate between them:

- ex2-meta-inf
- ex3-meta-inf

However, the `ex3-meta-inf` was not used at all so it is deleted in this PR. Then it seems no such confusing suffix is needed.